ctapipe/configuration/configuration.py

```diff
@@ -0,0 +1,312 @@
+'''
+https://docs.python.org/2/library/argparse.html
+'''
+
+
+from argparse import ArgumentParser
+from configparser import RawConfigParser
+from astropy.io import fits
+
+import numpy as np
+import sys
+
```

Comment:
Add `__all__` list with the classes / functions that are part of the "public API" and not implementation details.
Only if you do this, they will show up in the Sphinx docs.